frontend.py

```diff
@@ -44,7 +44,7 @@ def load_robots(state):
     robot_sprites = []
     for robot in state.robots:
         # Only alive and active robots will be drawn.
-        if robot.lives > 0 or not robot.inactive:
+        for robot in [robot for robot in state.robots if not robot.inactive]:
```

Comment:
Buď uprav tu podmínku, která tam byla nebo ten cyklus na řádku 45. 
Dva for cykly tady nedávají smysl. Momentálně se sprite vytvoří tolikrát, kolik je ve stavu robotů krát počet aktivních robotů. Pokud je 8 robotů ve stavu a z toho je 8 aktivních dostáváš 64 spritů místo 8.

Reply:
Takže jestli to chapu tak ten cyklus udelat jen pro aktivni roboty, tedy Radek 45 smažu a bude to cyklus  jako na novem 47?

Reply:
Jo 👍

Reply:
opraveno.